v1.1/certificate-rotation.md

```diff
@@ -0,0 +1,154 @@
+- Rotating the node and client certificates: CockroachDB generates new node and client certificates signed with the new CA certificate. The new node and client certificates are uploaded to the respective nodes and clients. The certs directory on the nodes is rescanned and the clients restarted to use the new certificates for verifying identities during inter-node and client-node communications. We highly recommend that you rotate the CA certificates in advance, and rotate the node and client certificates only when you are confident all the nodes and client have the combined CA uploaded to them. See [Why rotate CA certificate in advance](certificate-rotation.html#why-rotate-ca-certificates-in-advance) for detailed explanation.
+
+### Why CockroachDB generates a combined CA certificate
+
+After completing this process, the nodes have the new CA certificate, and the clients have the new CA certificates as and when they are restarted. But the nodes and client certificates are still signed with the old CA certificate. Thus the nodes and clients are unable to verify each other's identity during inter-node or client-node communication. 
+
+To overcome the issue, we take advantage of the fact that multiple CA certificates can be active at the same time. The node and client certificates can be signed with only the latest CA certificate, but while verifying the identity of another node or a client, they can check with multiple CA certificates uploaded to them. Thus instead of creating only the new certificate while rotating the CA certificates, CockroachDB creates a combined CA certificate that contains the new CA certificate followed by the old CA certificate. As and when node and client certificates are rotated, the combined CA certificate is used to verify old as well as new node and client certificates.
+ 
+### Why rotate CA certificates in advance 
+
+Going back to the CA certificate rotation process, the second step of the process involves rotating the node and client certificates. This involves signing the node certificates with new CA certificates. These new node certificates are uploaded to the nodes as soon as the certs directory on the node is rescanned. But the clients will get their new client certificates signed with the new CA certificates only when they are restarted. The clients won't even have the new CA certificates till they are restarted. Thus node certificates signed by the new CA certificate will not be accepted by clients that do not have the new CA certificate yet. To ensure all nodes and clients have the latest CA certificate, change CA certificates on a completely different schedule, for instance, months before changing the node and client certificates. 
```

Comment:
Instead of `These new node certificates are uploaded to the nodes ...`, it should be `These new nodes certificates are used by the nodes ...`